src/components/10-atoms/icon/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.0.0
+
+- BREAKING CHANGE: The new default for `size` of icons loaded by URL or SVG string is now 24x24 px instead of `auto`.
+- Functional icons should have only 3 sizes. You can now set these dimensions via `size` property. #2110
+- We renamed the size value `auto` to `original`.
```

Comment:
This is also a breaking change imho.

Reply:
No it is not, because: every string beside "small", "large" and "medium" causes the "auto"/original behaviour. --> oh i see a problem: "auto" is now the new default. thats wrong. will fix that.